.github/workflows/publish.yml

```diff
@@ -27,6 +27,11 @@ on:
         type: boolean
         default: true
         required: true
+      overwrite_if_exists:
```

Comment:
I don't think we want this - personally, I'd rather have ability to simply/easily copy an existing `publish` version to `latest` rather than re-publish a draft to `{version}` and `latest` folders.

And if something goes horribly wrong and a folder gets half-generated correctly, we can just delete it manually

Reply:
@fvankrieken, what if we chose incorrect draft build when publishing a version? Delete the generated version folder and republish since it wouldn't be a patch? Trying to think of our steps when a version must be overwritten

Reply:
Hmm. I agree that's a case that we should think about operationally. But it hasn't happened in my memory, and in that rare case, I think it's okay to "deviate" from our standard operations and delete a folder manually. I think my stance is stemming from feeling that having this checkbox seems to suggest that this option is "valid" in our standard workflow, which it really shouldn't be. So I'd rather err on side of caution in terms of devs not accidentally deviating from standard workflow in the case of a mis-publishing

Reply:
Had a typo in the last sentence which really ended up saying the opposite of what I meant - edited

Reply:
Per discussion, remove `overwrite` option.